crates/anvil/core/src/eth/transaction/mod.rs

```diff
@@ -4,6 +4,10 @@ use crate::eth::{
     receipt::Log,
     utils::{enveloped, to_revm_access_list},
 };
+#[cfg(feature = "impersonated-tx")]
+use alloy_primitives::U256 as rU256;
+#[cfg(feature = "impersonated-tx")]
+use alloy_rpc_types::Signature as AlloySignature;
```

Comment:
use fully qualified syntax where they're used instead of feature gating them here

Reply:
done in https://github.com/foundry-rs/foundry/pull/6767/commits/482ff026a3dce97a49e13b50bcc64424f19ad37b